### PR TITLE
Fill in start page content

### PR DESCRIPTION
**Pull request by tvararu**
https://trello.com/c/Wg9KgHKu

### After

image

### CODE REVIEW

**Review by felixclack — APPROVED** [review]
Very nice. One minor comment about the telephone number. Not a blocker though.

**Comment by tvararu** [comment]
@felixclack I'll merge this in and we can follow-up with a change to the `tel` pending the outcome of the discussion in the thread :+1:

File: app/views/pages/start.html.erb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    <h2 class="govuk-heading-m">If you need your TRN urgently</h2>
+
+    <p class="govuk-body">
+      Call the Teaching Regulation Agency, and have your National Insurance
+      number ready:
+    </p>
+
+    <ul class="govuk-list">
+      <li>Telephone: 020 7593 5394</li>
```
[the remark this answers is not in the document]
Comment (fofr) [reply app/views/pages/start.html.erb:36]: I think Safari (and other mobile browsers?) add these automatically and in contexts without software that can act on `tel:` the links are confusing. Which makes me thing they shouldn't be used. But I've asked in the x-gov Slack for more opinions.

[the remark this answers is not in the document]
Comment (fofr) [reply app/views/pages/start.html.erb:36]: We should not do this:
https://design-system.service.gov.uk/patterns/telephone-numbers/#do-not-display-telephone-numbers-as-links-on-devices-that-cannot-make-calls

[the remark this answers is not in the document]
Comment (tvararu) [reply app/views/pages/start.html.erb:36]: @fofr thanks for posting that, I was looking for it, I knew I must have seen clear guidance somewhere.

**Pull request merged**